common/test/acceptance/pages/lms/discussion.py

```diff
@@ -678,7 +678,7 @@ def is_browser_on_page(self):
         return self.q(css=".discussion-body section.home-header").present
 
     def perform_search(self, text="dummy"):
-        self.q(css=".forum-nav-search-input").fill(text + chr(10))
+        self.q(css="input.search").fill(text + chr(10))
```

Comment:
Nit: generally I don't like to have elements in selectors. Can this just be ``.search``?